data/json/items/tools.json

```diff
@@ -5790,6 +5790,30 @@
     "ammo": "NULL",
     "revert_to": "null",
     "use_action": "REMOVE_ALL_MODS",
+    "qualities": [["SCREW", 1]]
+  },
+
+  {
+    "id": "screwdriver_set",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "yellow",
+    "name": "screwdriver set",
+    "description": "This is a set of screwdrivers in several sizes and blade types.  Guaranteed to have the right tools for more precise work.",
```

Comment:
Woo, this is exactly where I was planning on going with high-quality screwdriver tools.
Now need same for spanners, and a mechanism for collecting enough misc screwdrivers to make a set :3